src/Homework_1/Matrix/TransformMatrix.cs

```diff
@@ -0,0 +1,71 @@
+namespace Matrices;
+
+/// <summary>
+/// Class that implements reading and writing matrix to/in file.
+/// </summary>
+public class TransformMatrix
+{
+    /// <summary>
+    /// Method that reads matrix from input file.
+    /// </summary>
+    /// <param name="filePath">Path to the file from which the matrix needs to be read.</param>
+    /// <returns>Matrix object.</returns>
+    public static Matrix ReadMatrixFromFile(string filePath)
+    {
+        if (!File.Exists(filePath))
+        {
+            throw new FileNotFoundException("No such file.");
+        }
+
+        if (new FileInfo(filePath).Length == 0)
+        {
+            throw new ArgumentException("File is empty.");
+        }
+
+        var matrixLines = new List<string>();
+
+        using var reader = new StreamReader(filePath);
+
+        string? line;
+        while ((line = reader.ReadLine()) != null)
+        {
+            if (line != "")
```

Comment:
```suggestion
            if (line != string.Empty)
```

Лучше использовать константы